recommender/recommender.py

```diff
@@ -235,98 +246,119 @@ def error_handler(self, error_msg, event, resource_id=None):
         tracker.emit(event, result)
         return result
 
+    def get_client_side_settings(self):
+        """
+        Return the parameters for client side environment setting.
+
+        Returns:
+                DISABLE_DEV_UX: whether to disable the UX under development
+                CURRENT_PAGE: the default page of resources showed to students
+                ENTRIES_PER_PAGE: the number of resources in each page
+                PAGE_SPAN: page range in pagination control
+                INTRO: whether to take users on a little tour when they see the RecommenderXBlock first time
```

Comment:
Shouldn't this be "whether to take the users on a little tour"? It doesn't matter if this is the first time they've seen the RecommenderXBlock, only what the database says.